index.js

```diff
@@ -0,0 +1,19 @@
+document.addEventListener("DOMContentLoaded", () => {
+  fetch("https://cors-anywhere.herokuapp.com/https://official-joke-api.appspot.com/jokes/ten").then(res => {
+    return res.json();
+}).then(jokes => {
+  let ul = document.querySelector('ul');
```

Comment:
Try to add the setup of the joke first, then find a way to add the punchline later.